python/cuml/tests/test_sklearn_compatibility.py

```diff
@@ -547,6 +547,11 @@
         # "check_transformers_unfitted": "UMAP does not raise error when transform called before fit",
         "check_parameters_default_constructible": "UMAP parameters are mutated on init",
         "check_fit_check_is_fitted": "UMAP passes check_is_fitted before being fit",
+        "check_estimators_empty_data_messages": "UMAP does not handle empty data",
```

Comment:
How much work would it be to make it so that the test passes instead of xfailing it? We already raise the right type of exception, is it "just" a question of adjusting the text to match what the check expects?